### PR TITLE
Add workspace setting to ignore specific project folder in multi-root workspace

### PR DESCRIPTION
**Pull request by EzioLi01**

### CODE REVIEW

**Review by benjaminbi — APPROVED** [review]

**Comment by AttahPepeChilee** [comment]
add-settings-ignore-specific-folder1

File: package.json
Comment (AttahPepeChilee) [inline package.json:1]: master

Comment (AttahPepeChilee) [reply package.json:1]: add-settings-ignore-specific-folder1

File: package.nls.json
Comment (AttahPepeChilee) [inline package.nls.json:1]: package.nls.json

**Review by AttahPepeChilee — COMMENTED** [review]
add-settings-ignore-specific-folder1

**Comment by AttahPepeChilee** [comment]
[](url)add-settings-ignore-specific-folder1

**Pull request merged**